src/cephadm/cephadm

```diff
@@ -5109,6 +5440,15 @@ def _get_parser():
         '--fsid',
         help='cluster FSID')
 
+    parser_gather_facts = subparsers.add_parser(
+        'gather-facts', help='gather and return host related information')
+    parser_gather_facts.set_defaults(func=command_gather_facts)
+    parser_gather_facts.add_argument(
+        '--fact-format',
+        choices=['json'],
+        default='json',
+        help='format for the output')
```

Comment:
I'd like to omit this here. And instead make use of mgr/cephadm to properly generate a human friendly output.

Reply:
ACK - dropped the option